example/timings/time_forest_partition.cxx

```diff
@@ -241,13 +259,21 @@ t8_time_forest_cmesh_mshfile (t8_cmesh_t cmesh, const char *vtu_prefix,
       if (r == refine_rounds - 1) {
         t8_forest_set_profiling (forest_partition, 1);
         if (do_ghost) {
+          t8_debugf
+            ("[D] Use fast ghost, as soon as nca is implemented for pyramid\n");
```

Comment:
Is this still valid?

Reply:
Yes, the NCA is not implemented yet. But I should implement it, see Issue #184